lib/evaluator/handlers.js

```diff
@@ -81,16 +81,47 @@ exports.FilterExpression = function(ast) {
  * @private
  */
 exports.Identifier = function(ast) {
+  function contextHasProp(context, prop) {
```

Comment:
Why is this defined in two places?

Reply:
The other place is in the built artifact. :-)